src/main/java/seedu/address/model/Model.java

```diff
@@ -6,13 +6,17 @@
 import javafx.collections.ObservableList;
 import seedu.address.commons.core.GuiSettings;
 import seedu.address.model.person.Contact;
+import seedu.address.model.person.Type;
 
 /**
  * The API of the Model component.
  */
 public interface Model {
     /** {@code Predicate} that always evaluate to true */
     Predicate<Contact> PREDICATE_SHOW_ALL_CONTACTS = unused -> true;
```

Comment:
Suggestion for consistency - makes it clear that the contact is the value that's intentionally not checked.
```suggestion
    Predicate<Contact> PREDICATE_SHOW_ALL_CONTACTS = contact -> true;
```

Reply:
Done.